build.gradle

```diff
@@ -190,3 +193,25 @@ task ktlintFormat(type: JavaExec, group: "formatting") {
     main = "com.github.shyiko.ktlint.Main"
     args "-F", "${projectDir}/components/**/*.kt", "${projectDir}/gradle-plugin/**/*.kt", "buildSrc/**/*.kt", "!**/build"
 }
+
+task docs(type: org.jetbrains.dokka.gradle.DokkaAndroidTask, overwrite: true) {
```

Comment:
Since this is platform specific, maybe it makes more sense for it (and the other dependencies) to live in [here](https://github.com/mozilla/glean/blob/master/glean-core/android/build.gradle)?